test/generate-testcase.js

```diff
@@ -0,0 +1,117 @@
+var path = require("path");
+var fs = require("fs");
+var jsdom = require("jsdom").jsdom;
+var serializeDocument = require("jsdom").serializeDocument;
+var http = require("http");
+var vm = require("vm");
+var readabilityPath = path.join(__dirname, "..", "Readability.js");
+var jsdomPath = path.join(__dirname, "..", "JSDOMParser.js");
+var ctx = {};
+ctx.dump = console.log
+ctx.console = console;
+vm.runInNewContext(fs.readFileSync(jsdomPath), ctx, jsdomPath);
+vm.runInNewContext(fs.readFileSync(readabilityPath), ctx, readabilityPath);
+var Readability = ctx.Readability;
+var JSDOMParser = ctx.JSDOMParser;
+
+
+if (process.argv.length < 3) {
+  console.log("Need at least a destination slug and potentially a URL (if the slug doesn't have source).");
+  return 0;
+}
+
+var slug = process.argv[2];
+var url = process.argv[3];
+
+var destRoot = path.join(__dirname, "test-pages", slug);
+
+fs.mkdir(destRoot, function(err) {
```

Comment:
Someone downvoted me for suggesting it on stackoverflow a long time ago. It still hurts. ;-)